src/textblob/base.py

```diff
@@ -19,11 +25,11 @@ class BaseTagger(metaclass=ABCMeta):
     """
 
     @abstractmethod
-    def tag(self, text, tokenize=True):
+    def tag(self, text: str, tokenize=True) -> list[tuple[str, str]]:
         """Return a list of tuples of the form (word, tag)
         for a given set of text or BaseBlob instance.
         """
-        return
+        raise NotImplementedError("Subclass must implement a tag method")
```

Comment:
nit: i generally use the `Ellipsis` literal for abstract methods [like the python docs do](https://docs.python.org/3/library/abc.html#abc.abstractmethod) (thought the python docs aren't always consistent with this)

```suggestion
        ...
```

Reply:
Ah, good call. That's cleaner